english/bob-barker/2050s/2050s.md

```diff
@@ -0,0 +1,9 @@
+Your eyes flash open.  You realize you are late to something, but you are not sure what that something is.  You look down at your wrist and see your iWrist flash red.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.